pkg/controller/gittrack/handler.go

```diff
@@ -78,6 +79,13 @@ func (h handlerResult) asMetricOpts(repository string) *metricsOpts {
 func (r *ReconcileGitTrack) handleGitTrack(gt farosv1alpha1.GitTrackInterface) handlerResult {
 	var result handlerResult
 
+	// don't reconcile if it's a gittrack and they're disabled
+	// we shouldn't ever get here because reconcile should be turned off at the controller level,
+	// but for testing and safeguarding, check anyway
+	if _, ok := gt.(*farosv1alpha1.ClusterGitTrack); ok && r.gitTrackMode == farosflags.GTMDisabled {
```

Comment:
I don't think we should perform this evaluation, given it is going to happen for every reconcile and we don't have it's counterpart for ClusterGitTracks, we should just test and rely on the controller level off switches